docs/src/pages/customization/themes/themes.md

```diff
@@ -289,6 +289,12 @@ html {
 
 {{"demo": "pages/customization/themes/FontSizeTheme.js"}}
 
+#### Responsive fonts
+
+You can use the new "@material-ui/csskit" that at the moment contains a helper function to configure Typography with responsive fonts.
```

Comment:
```suggestion
You can use the `responsiveTypography` helper in `@material-ui/csskit` to make Typography font sizes responsive to the window size.
```
Still seems a bit off to me. I would wait for more opinions.

Reply:
done, as per https://github.com/mui-org/material-ui/pull/14573#pullrequestreview-217995134 too